donjo-app/views/web/mandiri/footer_mandiri.php

```diff
@@ -19,43 +19,5 @@
       </div>
     </div>
 
-    <!-- jQuery 3 -->
```

Comment:
Mengapa javascript dipindah ke header? Kita berusaha meletakkan javascript to bagian akhir html, sesuai anjuran praktek baik:
- https://www.duniailkom.com/tutorial-belajar-javascript-posisi-terbaik-meletakkan-kode-javascipt-di-dalam-html/
- https://books.google.com.au/books?id=ZrRCAwAAQBAJ&pg=PT160&dq=is+it+best+to+put+javascript+at+the+end+of+html&hl=en&sa=X&ved=0ahUKEwiav7fds67lAhUCSX0KHWx1BrsQ6AEIRTAE#v=onepage&q=is%20it%20best%20to%20put%20javascript%20at%20the%20end%20of%20html&f=false

Mohon dikembalikan dan ditinjau kembali mengapa anda perlukan.